hq_superset/views.py

```diff
@@ -40,6 +40,8 @@
 
 
 class HQDatasourceView(BaseSupersetView):
+    class_permission_name = "ExploreFormDataRestApi"
```

Comment:
just checking on why was this added?

Reply:
This was to ensure that only users with edit permissions can import data sources.